Automattic/jetpack
# Fix/waf activation

#### Changes proposed in this Pull Request:

* Adds a new activation/deactivation endpoint to the WAF package.
* Calls those endpoints when the module activation is toggled to ensure it's setup correctly

#### Jetpack product discussion


p1651152931717599-slack-C029WFNV69M
p1651157354161659-slack-C029WFNV69M

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the Jetpack settings page.
* Activate the Waf module.
* Verify that the jetpack_waf_mode wp_option was inserted correctly and that the bootstrap file was generated.
* Deactivate the module and verify that the wp_option is deleted.
